ParseTask2/ParseTask2.BL/Helpers/ConfigHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+namespace ParseTask2.BL.Helpers
+{
+    public static class ConfigHelper
+    {
+        public static readonly string BaseAddress = @"http://swapi.co/api/";
+        public static readonly string AcceptHeader = "application/json";
```

Comment:
Best place for such strings is web.config. Please read some articles about storing and reading values from it